fv3core/fv3core/stencils/remapping.py

```diff
@@ -433,39 +510,39 @@ def __call__(
         nq: int,
     ):
         """
-        pt: D-grid potential temperature (inout)
-        delp: Pressure Thickness (inout)
-        delz: Vertical thickness of atmosphere layers (in)
-        peln: Logarithm of interface pressure (inout)
-        u: D-grid x-velocity (inout)
-        v: D-grid y-velocity (inout)
-        w: Vertical velocity (inout)
-        ua: A-grid x-velocity (inout)
-        va: A-grid y-velocity (inout)
-        cappa: Power to raise pressure to (inout)
-        q_con: Total condensate mixing ratio (inout)
-        q_cld: Cloud fraction (inout)
-        pkz: Layer mean pressure raised to the power of Kappa (in)
-        pk: Interface pressure raised to power of kappa, final acoustic value (inout)
-        pe: Pressure at layer edges (inout)
-        hs: Surface geopotential (in)
-        te0_2d: Atmosphere total energy in columns (inout)
-        ps: Surface pressure (inout)
-        wsd: Vertical velocity of the lowest level (in)
-        omga: Vertical pressure velocity (inout)
-        ak: Atmosphere hybrid a coordinate (Pa) (in)
-        bk: Atmosphere hybrid b coordinate (dimensionless) (in)
-        pfull: Pressure full levels (in)
-        dp1: Pressure thickness before dyn_core (inout)
-        ptop: The pressure level at the top of atmosphere (in)
-        akap: Poisson constant (KAPPA) (in)
-        zvir: Constant (Rv/Rd-1) (in)
-        last_step: Flag for the last step of k-split remapping (in)
-        consv_te: If True, conserve total energy (in)
-        mdt : Remap time step (in)
-        bdt: Timestep (in)
-        do_adiabatic_init: If True, do adiabatic dynamics (in)
-        nq: Number of tracers (in)
+        pt (inout): D-grid potential temperature
+        delp (inout): Pressure Thickness
+        delz (in): Vertical thickness of atmosphere layers
+        peln (inout): Logarithm of interface pressure
+        u (inout): D-grid x-velocity
+        v (inout): D-grid y-velocity
+        w (inout): Vertical velocity
+        ua (inout): A-grid x-velocity
+        va (inout): A-grid y-velocity
+        cappa (inout): Power to raise pressure to
+        q_con (inout): Total condensate mixing ratio
+        q_cld (inout): Cloud fraction
+        pkz (in): Layer mean pressure raised to the power of Kappa
+        pk (inout): Interface pressure raised to power of kappa, final acoustic value
+        pe (inout): Pressure at layer edges
+        hs (in): Surface geopotential
+        te0_2d (inout): Atmosphere total energy in columns
+        ps (inout): Surface pressure
+        wsd (in): Vertical velocity of the lowest level
+        omga (inout): Vertical pressure velocity
+        ak (in): Atmosphere hybrid a coordinate (Pa)
+        bk (in): Atmosphere hybrid b coordinate (dimensionless)
+        pfull (in): Pressure full levels
+        dp1 (inout): Pressure thickness before dyn_core
```

Comment:
```suggestion
        dp1 (inout): Pressure thickness before dyn_core (only written if do_sat_adjust=True)
```
It looks like saturation_adjustment's first field is `out`, which is the only time dp1 is used.